source/jormungandr/jormungandr/scenarios/helper_classes/streetnetwork_path.py

```diff
@@ -80,27 +93,105 @@ def __init__(
         self._request_id = request_id
         self._async_request()
 
-    @new_relic.distributedEvent("direct_path", "street_network")
-    def _direct_path_with_fp(self):
-        with timed_logger(self._logger, 'direct_path_calling_external_service', self._request_id):
-            try:
-                return self._streetnetwork_service.direct_path_with_fp(
+    def poi_to_pt_object(self, poi):
+        return type_pb2.PtObject(poi=poi, uri=poi.uri, embedded_type=type_pb2.POI, name=poi.name)
+
+    def get_pt_objects(self, entry_point):
+        if include_poi_access_points(self._request, entry_point, self._mode):
+            for o in entry_point.poi.children:
+                yield self.poi_to_pt_object(o)
```

Comment:
```suggestion
            for o in entry_point.poi.children:
                yield self.poi_to_pt_object(o)
            return
```
otherwise, the entry_point will also be included, which is not what you want

Reply:
Done